src/main/scala/org/broadinstitute/dsde/workbench/sam/service/PolicyEvaluatorService.scala

```diff
@@ -13,6 +14,20 @@ class PolicyEvaluatorService(
     private val resourceTypes: Map[ResourceTypeName, ResourceType],
     private val accessPolicyDAO: AccessPolicyDAO)(implicit val executionContext: ExecutionContext)
     extends LazyLogging {
+  def initPolicy(): IO[Unit] = {
+    val policyName = AccessPolicyName("admin-notifier-set-public")
+    accessPolicyDAO.createPolicy(AccessPolicy(
+      FullyQualifiedPolicyId(FullyQualifiedResourceId(SamResourceTypes.resourceTypeAdminName, ResourceId("managed-group")), policyName),
+      Set.empty, //TODO: question: what should memebers be?
+      WorkbenchEmail("dummy@gmail.com"),
```

Comment:
@dvoet I think to do that we'll need to pass `emailDomain` in if I understand correctly...Since I did some refactoring about configurations in https://github.com/broadinstitute/sam/pull/230, I think it'll be a bit cleaner if we do that after that.